client/question/services/SolutionHandlerServiceSpec.js

```diff
@@ -454,8 +456,18 @@ describe('SolutionHandlerService', function() {
           SolutionHandlerService.processSolutionAsync(
             orderedTasks, starterCode, studentCode, auxiliaryCode, 'python'
           ).then(function(feedback) {
-            expect(feedback.getParagraphs()[0].getContent()).toBe(
-              'Your code produced the following result:');
+            var expectedFeedbackTextResult = 'valid feedback text';
+            var actualFeedbackTextResult = 'invalid feedback text';
+            var actualFeedbackText = feedback.getParagraphs()[0].getContent();
```

Comment:
Don't do this -- the type should be deterministic, and it should be possible to just do something like `expect(CORRECTNESS_FEEDBACK_TEXT[...].includes(actualFeedbackTextResult)).toBe(true)`.

Reply:
The way it's set up, it needs to traverse multiple feedback type arrays. Modified it a bit, but please let me know if there is a better way.

Reply:
I don't understand the need to traverse the multiple arrays. Shouldn't the expected typeKey be deterministic (for this particular input)? I can understand the uncertainty in the .includes() part, since we're randomly generating that, but I would expect the typeKey to be deterministic.

Reply:
OK, I think I changed it they way you are describing.